base_partner_sequence/__openerp__.py

```diff
@@ -21,34 +21,19 @@
 #
 ##############################################################################
```

Comment:
Please use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py